app/templates/keyboard_shortcuts.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div id="accordion">
+    <div class="card">
+      <div class="card-header" id="General">
+        <h5 class="mb-0">
+          <button class="btn btn-link" data-toggle="collapse" data-target="#GeneralShortcuts"
+                  aria-expanded="true" aria-controls="collapseOne">
+            General Shortcuts - Navigation
+          </button>
+        </h5>
+      </div>
+      <div class="collapse" id="GeneralShortcuts">
```

Comment:
id names should be lowercase and separated by "-".

Reply:
Thanks! done :)